README.md

```diff
@@ -90,6 +92,21 @@ test('esmock.strict mocks', async () => {
 })
 ```
 
+TypeScript
+``` typescript
+import type MinecraftWorld from '../src/minecraftWorld.js';
```

Comment:
Include all of imports needed for the example, and separate the imports from the first test with an empty newline. Remove the dangling comma after `'HARD',`
```typescript
import test from 'node:test'
import assert from 'node:assert'
import esmock from 'esmock'
import type MinecraftWorld from '../src/minecraftWorld.js'

test('allows you to specify the type of returned exports', async () => {
```

Reply:
> Remove the dangling comma after 'HARD',

This is actually a best practice, see [here](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma).